### PR TITLE
reapply filters after duplication of cohort-selection

### PR DESCRIPTION
**Pull request by beaugogh**

### CODE REVIEW

File: src/app/components/cohort-selection/cohort-selection.directive.js [hunk 1]
```diff
@@ -30,13 +30,13 @@ angular.module('transmartBaseUi')
                     /*
                      * 1. copy and apply history of one workspace to another
                      */
-                    // box.ctrl.history = _.cloneDeep(box.duplication.ctrl.history);
```
Comment (rnugraha) [inline src/app/components/cohort-selection/cohort-selection.directive.js:33]: Are these code going to be used in the future? if not pls remove

File: src/app/components/cohort-selection/cohort-selection.controller.js [hunk 2]
```diff
@@ -940,23 +941,34 @@ angular.module('transmartBaseUi')
                         }
                     });
 
-                    if(removed.length > 0) {
+                    if (removed.length > 0) {
```
Comment (rnugraha) [inline src/app/components/cohort-selection/cohort-selection.controller.js]: `return removed.length ? true : false;`

(just suggestion)

File: src/app/components/cohort-selection/cohort-selection.directive.js (hunk 1 above)
Comment (beaugogh) [reply src/app/components/cohort-selection/cohort-selection.directive.js:33]: ward mentioned that history management would be useful in future

File: src/app/components/cohort-selection/cohort-selection.controller.js (hunk 2 above)
Comment (beaugogh) [reply src/app/components/cohort-selection/cohort-selection.controller.js]: good point, will change it

File: src/app/components/cohort-selection/cohort-selection.controller.js [hunk 3]
```diff
@@ -941,12 +941,7 @@ angular.module('transmartBaseUi')
                         }
                     });
 
-                    if (removed.length > 0) {
-                        return true;
-                    }
-                    else {
-                        return false;
-                    }
+                    return removed.length ? true: false;
```
Comment (kretep) [inline src/app/components/cohort-selection/cohort-selection.controller.js:944]: How about 'return removed.length > 0;'? :)

Comment (beaugogh) [reply src/app/components/cohort-selection/cohort-selection.controller.js:944]: they are equivalent :)

Comment (kretep) [reply src/app/components/cohort-selection/cohort-selection.controller.js:944]: Are they? It saves you one check and two literals.

Comment (kretep) [reply src/app/components/cohort-selection/cohort-selection.controller.js:944]: Well, maybe one. Doesn't matter really, it's fine like this

**Pull request merged**